pages/cart/cart.css

```diff
@@ -0,0 +1,100 @@
+:root{
+    --secondary-color: black;
+    --gray: gray;
+}
+.card-and-price-details {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 25rem;
+}
+.cart-card {
+  display: flex;
+  border: 1px solid var(--secondary-color);
+  width: 26rem;
+  text-transform: capitalize;
+  box-shadow: 5px 5px var(--gray);
+}
+.cart-products-heading{
+    text-align: center;
+    margin-top: 1rem;
+}
+.cart-card-image {
+  width: 13rem;
+}
+.cart-card-details {
+  padding: 1rem;
+}
+.cart-card-details-head {
+  font-weight: light;
+  margin-bottom: 1rem;
+}
+.cart-card-details-price {
+  display: flex;
+  margin-bottom: 1rem;
+}
+.cart-card-details-price-discount,.price-details-heading,.price {
+  font-size: 1.1rem;
+  font-weight: bolder;
+}
+.cart-card-details-price-real {
+  margin-left: 1rem;
+  text-decoration: line-through;
+}
```

Comment:
try to limit the class names to two or three like name1-name2-name3 from next time ,it's 5 here.

Reply:
sure will do that